Hybrid adapter. Added support In-Image format

hybrid-ai opened the pull request:
Added support In-Image format. PR docs https://github.com/prebid/prebid.github.io/pull/2341

aleksatr commented:
Hi @hybrid-ai , `hybridBidAdapter.js` unit test coverage is below the minimum 80%, would you please add more tests to bring the coverage up to at least 80%?

hybrid-ai commented:
> Hi @hybrid-ai , `hybridBidAdapter.js` unit test coverage is below the minimum 80%, would you please add more tests to bring the coverage up to at least 80%?
> 
> To generate and view the code coverage reports:
> 
> ```shell
> gulp test-coverage
> gulp view-coverage
> ```

hybrid-ai commented:
> Hi @hybrid-ai , `hybridBidAdapter.js` unit test coverage is below the minimum 80%, would you please add more tests to bring the coverage up to at least 80%?
> 
> To generate and view the code coverage reports:
> 
> ```shell
> gulp test-coverage
> gulp view-coverage
> ```

Thank you for your comment. We fixed it.

aleksatr commented:
> > Hi @hybrid-ai , `hybridBidAdapter.js` unit test coverage is below the minimum 80%, would you please add more tests to bring the coverage up to at least 80%?
> > To generate and view the code coverage reports:
> > ```shell
> > gulp test-coverage
> > gulp view-coverage
> > ```
> 
> Hi, @aleksatr thank you for your comment. We fixed it.

Hi @hybrid-ai , check your ci/circleci build output, it shows lint errors:

```
/home/circleci/Prebid.js/test/spec/modules/hybridBidAdapter_spec.js
  31:15  error  Strings must use singlequote  quotes
  35:20  error  Expected consistent spacing   standard/object-curly-even-spacing
```

hybrid-ai commented:
> > > Hi @hybrid-ai , `hybridBidAdapter.js` unit test coverage is below the minimum 80%, would you please add more tests to bring the coverage up to at least 80%?
> > > To generate and view the code coverage reports:
> > > ```shell
> > > gulp test-coverage
> > > gulp view-coverage
> > > ```
> > 
> > 
> > Hi, @aleksatr thank you for your comment. We fixed it.
> 
> Hi @hybrid-ai , check your ci/circleci build output, it shows lint errors:
> 
> ```
> /home/circleci/Prebid.js/test/spec/modules/hybridBidAdapter_spec.js
>   31:15  error  Strings must use singlequote  quotes
>   35:20  error  Expected consistent spacing   standard/object-curly-even-spacing
> ```

Fixed

aleksatr commented on modules/hybridBidAdapter.js:
```diff
@@ -94,6 +97,35 @@ function buildBid(bidData) {
+    actionUrls.stopAnimationUrls = bidData.inImage.stopanimationtrackers || [];
+    actionUrls.closeBannerUrls = bidData.inImage.closebannertrackers || [];
+
+    bidData.inImage.but = true;
+
+    if (bidData.inImage.but) {
```
@hybrid-ai what is the purpose of this `if` when you set this param to true on L116?

hybrid-ai replied:
@aleksatr This is an additional button for inImage banner and that button has its own styles

aleksatr replied:
@hybrid-ai , ok, but you set `bidData.inImage.but` to true on L116, so `if` statement on L118 is not needed, it will always evaluate to `true`

hybrid-ai replied:
@aleksatr I am sorry. I added it for debug but I forget to delete this line. Thank you for your comment.

aleksatr commented:
LGTM
